### DISCUSSION TITLE
Does it support Kafka and Clickhouse?

### DISCUSSION THREAD

**Discussion by enact-on** [umami-software/umami#2612] [Q&A]
Hello,

I see references to Kafka and Clickhouse at many different places like source code, discussion, issues, and in the evn file.

But I don't see any documentation for these. Are they supported or not?
If yes, how can i configure to use them?

Thanks.

**Comment by tungtouch** [comment]
same question

**Comment by Pandoks** [comment]
Only thing I found was:
https://boehs.org/node/umami-clickhouse
